nanoFramework.Device.Can/nanoFramework.Device.Can.nfproj

```diff
@@ -71,11 +72,11 @@
       <ProjectConfigurationsDeclaredAsItems />
     </ProjectCapabilities>
   </ProjectExtensions>
-  <Import Project="..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets')" />
+  <Import Project="..\packages\Nerdbank.GitVersioning.3.5.113\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.5.113\build\Nerdbank.GitVersioning.targets')" />
```

Comment:
nbgv 3.5.113  has both props and targets... I think you may have updated this manually... better let VS or nuget CLI do it 😉

Reply:
TBH, I removed the props, as was fire fighting... I will re-add it!